Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -231,79 +231,82 @@ namespace Multiplayer
-                byteStream.Write(assetHint.size(), assetHint.data());
-                AZ::Utils::SaveObjectToStream(byteStream, AZ::DataStream::ST_BINARY, asset.GetData(), asset.GetData()->GetType());
-            }
+        // Serialize Asset information and AssetData into a potentially large buffer
+        for (const auto& asset : assetData)
+        {
+            AZ::Data::AssetId assetId = asset.GetId();
+            AZStd::string assetHint = asset.GetHint();
+            uint32_t hintSize = aznumeric_cast<uint32_t>(assetHint.size());
 
-            const AZ::CVarFixedString remoteAddress = editorsv_serveraddr;
-            if (editorsv_launch)
+            byteStream.Write(sizeof(AZ::Data::AssetId), reinterpret_cast<void*>(&assetId));
+            byteStream.Write(sizeof(uint32_t), reinterpret_cast<void*>(&hintSize));
+            byteStream.Write(assetHint.size(), assetHint.data());
+            AZ::Utils::SaveObjectToStream(byteStream, AZ::DataStream::ST_BINARY, asset.GetData(), asset.GetData()->GetType());
+        }
+
+        const AZ::CVarFixedString remoteAddress = editorsv_serveraddr;
+        if (editorsv_launch)
+        {
+            if (LocalHost != remoteAddress)
             {
-                if (LocalHost != remoteAddress)
-                {
-                    AZ_Warning(
-                        "MultiplayerEditor", false,
-                        "Launching EditorServer skipped because incompatible cvars. editorsv_launch=true, meaning you want to launch an editor-server on this machine, but the editorsv_serveraddr is %s instead of the local address (127.0.0.1). "
-                        "Please either set editorsv_launch=false and keep the remote editor-server, or set editorsv_launch=true and editorsv_serveraddr=127.0.0.1.",
-                        remoteAddress.c_str())
-                    return;
-                }
-                
-                // Begin listening for MPEditor packets before we launch the editor-server.
-                // The editor-server will send us (the editor) an "EditorServerReadyForLevelData" packet to let us know it's ready to receive data.
-                INetworkInterface* editorNetworkInterface =
-                    AZ::Interface<INetworking>::Get()->RetrieveNetworkInterface(AZ::Name(MpEditorInterfaceName));
-                AZ_Assert(editorNetworkInterface, "MP Editor Network Interface was unregistered before Editor could connect.");
-                editorNetworkInterface->Listen(editorsv_port);
-
-                // Launch the editor-server
-                m_serverProcess = LaunchEditorServer();
+                AZ_Warning(
+                    "MultiplayerEditor", false,
+                    "Launching EditorServer skipped because incompatible cvars. editorsv_launch=true, meaning you want to launch an editor-server on this machine, but the editorsv_serveraddr is %s instead of the local address (127.0.0.1). "
```

Comment:
Why do we not ignore editorsv_serveraddr in this case? 

If not
"Launching editor server skipped because of incompatible settings. When using editorsv_launch=true, editorsv_serveraddr must be set to local address (127.0.0.1) instead %s"

Reply:
I haven't actually changed this code, only whitespace changes. 
Shouldn't be a problem to ignore editorsv_serveraddr indeed but I'll leave it to a separate PR since it needs more testing/validating.
Changed the message to the one you suggested, thanks!